src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData.Web/Components/Layout/SurveyPrompt.razor

```diff
@@ -0,0 +1,20 @@
+<div class="surveyContainer" role="alert">
+    <span class="tool-icon" aria-hidden="true">
+        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="size-5">
+            <path stroke-linecap="round" stroke-linejoin="round" d="M16.862 4.487l1.687-1.688a1.875 1.875 0 112.652 2.652L10.582 16.07a4.5 4.5 0 01-1.897 1.13L6 18l.8-2.685a4.5 4.5 0 011.13-1.897l8.932-8.931zm0 0L19.5 7.125" />
+        </svg>
+    </span>
+
+    <span>
+        <strong>@Title</strong>
+        Please take our
+        <a target="_blank" href="https://aka.ms/dotnet-chat-template-survey">brief survey</a>
+        and tell us what you think.
+    </span>
+</div>
+
+@code {
+    // Demonstrates how a parent component can supply parameters
```

Comment:
Demonstrates to whom? Isn't this whole page temporary?

Reply:
Hold over from the original Blazor template where I got this. I've removed it now.